code/acting/src/acting/pure_pursuit_controller.py

```diff
@@ -161,13 +163,16 @@ def __calculate_steer(self) -> float:
         :return:
         """
         l_vehicle = 2.85  # wheelbase
-        k_ld = 2.0  # todo: tune
-        look_ahead_dist = 5.0  # offset so that ld is never zero
-
-        if round(self.__velocity, 1) < 0.1:
-            look_ahead_dist += 1.0
+        k_ld = 1.0  # todo: tune
```

Comment:
Is this tuned now or does it need further work?

Reply:
Should be good enough for now, fixed in [d556b8e](https://github.com/ll7/paf22/pull/210/commits/d556b8ea0a8a784c0dcf05d11e1a526768c9dfce)